project/docs/gen_credits.py.jinja

```diff
@@ -23,6 +23,7 @@ def get_credits_data() -> dict:
     metadata_pdm = toml.load(project_dir / "pyproject.toml")["tool"]["pdm"]
     lock_data = toml.load(project_dir / "pdm.lock")
     project_name = metadata["name"]
+    more_credits = metadata.get("more_credits", "")
```

Comment:
This is not correct, there's no `more_credits` key to define in pyproject.toml.

```suggestion
```

Reply:
Oh I see, I though you wanted to pass the `more_credits` through the pyproject.toml `project` section as a new argument of the copier-pdm project.  🙂